src/Workspaces/Core/Portable/SymbolKey/SymbolKey.ErrorTypeSymbolKey.cs

```diff
@@ -57,6 +88,28 @@ public static SymbolKeyResolution Resolve(SymbolKeyReader reader)
                 return CreateResolution(result);
             }
 
+            private static SymbolKeyResolution ResolveContainer(SymbolKeyReader reader)
+            {
+                var type = reader.ReadInteger();
+                switch (type)
+                {
+                    case 0:
+                        return reader.ReadSymbolKey();
+                    case 1:
+                        var namespaceNames = reader.ReadStringArray();
+                        var currentNamespace = reader.Compilation.GlobalNamespace;
+
+                        foreach (var name in namespaceNames)
```

Comment:
Doesn't namespaceNames need to be reversed? GetContainingNamespaceNames is recording them innermost namespace to outermost namespace, right?

Reply:
This may need a test for a type that's not parented by "System" either way, since right now your additional test won't hit this bug nor prove it's non-existence.

Reply:
you're likely write.  i def thought about that, but don't know if i did anything.  i'll create a test with a deeper namespace.  good catch.

Reply:
this was a bug and test caught it.  thanks!